Program/commandline.h

```diff
@@ -77,6 +77,10 @@ class CommandLine
 					ap.nbClose = atoi(argv[i+1]);
 				else if (std::string(argv[i]) == "-targetFeasible")
```

Comment:
You may need to increase the value set at line https://github.com/vidalt/HGS-CVRP/blob/cc9c54fdf005dc1576ac34e044a7a55c18d705ec/Program/commandline.h#L45 to make sure we could specify all available parameters without running into an exception